tests/acceptance/user-can-create-project-with-map-test.js

```diff
@@ -4,51 +4,66 @@ import {
   currentURL,
   click,
   fillIn,
+  settled,
 } from '@ember/test-helpers';
 import { setupApplicationTest } from 'ember-qunit';
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
-import { faker } from 'ember-cli-mirage';
-import random from 'labs-applicant-maps/tests/helpers/random-geometry';
-import LabsLayers from 'labs-applicant-maps/components/labs-layers';
-import DrawMode from 'labs-applicant-maps/components/project-geometries/modes/draw';
-import setupMapMocks from 'labs-applicant-maps/tests/helpers/setup-map-mocks';
-
-const { randomPolygon } = random;
+import setupMapMocks from 'labs-applicant-maps/tests/helpers/mapbox-gl-stub';
+import setupComposerMocks from 'labs-applicant-maps/tests/helpers/mapbox-composer-stub';
 
 module('Acceptance | user can create project with map', function(hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
   setupMapMocks(hooks);
+  setupComposerMocks(hooks);
 
-  hooks.beforeEach(async function() {
-    this.server.createList('layer-group', 10);
-    this.server.create('layer-group', { id: 'tax-lots' });
-
-    let onLayerClick;
-    this.owner.register('component:labs-layers', LabsLayers.extend({
-      init(...args) {
-        this._super(...args);
-
-        onLayerClick = this.get('onLayerClick');
+  test('User can create new project', async function(assert) {
+    const triangle = {
+      type: 'Feature',
+      properties: {
+        id: 1,
+        label: 'test',
       },
-      'data-test-labs-layers': true,
-      click() {
-        const randomFeature = randomPolygon(1).features[0];
-        randomFeature.properties.bbl = faker.random.uuid();
-        onLayerClick(randomFeature);
+      geometry: {
+        type: 'Polygon',
+        coordinates: [[[1, 1], [0, 1], [1, 0], [1, 1]]],
       },
-    }));
-
-    this.owner.register('component:project-geometries/modes/draw', DrawMode.extend({
-      'data-test-draw-mock': true,
-      click() {
-        const randomFeatures = randomPolygon(1, true);
-        this.set('geometricProperty', randomFeatures);
+      layer: {
+        id: 'test',
       },
-    }));
-  });
+    };
+
+    const triangleFC = {
+      type: 'FeatureCollection',
+      features: [triangle],
+    };
+
+    let currentReturnValue = {
```

Comment:
@julialucyhogan good point, I actually think this the better approach because it keeps the stub data closer to the test itself. 

The reason this is scoped only to the this test is that there is only one test in this file.  In the other test, I set that information on `this` because i needed it available across multiple tests (it was handled in a beforeEach). I don't like that approach as much because it's more "mystery guest" behavior in a test.